packages/devtools_app/lib/src/app_size/file_import_container.dart

```diff
@@ -138,18 +140,35 @@ class _FileImportContainerState extends State<FileImportContainer> {
     );
   }
 
-// TODO(kenz): uncomment once file picker support is added
-//  Widget _buildImportButton() {
-//    return Row(
-//      mainAxisAlignment: MainAxisAlignment.center,
-//      children: [
-//        OutlinedButton(
-//          onPressed: () {},
-//          child: const MaterialIconLabel(Icons.file_upload, 'Import File'),
-//        ),
-//      ],
-//    );
-//  }
+  Widget _buildImportButton() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        OutlinedButton(
+          onPressed: () async {
+            final acceptedTypeGroups = [
+              XTypeGroup(extensions: ['.json']),
+            ];
+            final file = await openFile(acceptedTypeGroups: acceptedTypeGroups);
+            final json = jsonDecode(await file.readAsString());
+            final devToolsJsonFile = DevToolsJsonFile(
+              name: file.name,
+              lastModifiedTime: await file.lastModified(),
+              data: json,
+            );
+            _handleImportedFile(devToolsJsonFile);
+          },
+          child: const MaterialIconLabel(Icons.file_upload, 'Import File'),
+        ),
+      ],
+    );
+  }
+
+  // Only show the import button in linux, macOS and web.
+  bool get _shouldShowImportButton =>
```

Comment:
can this just be true? what will happen if someone tries to build devtools and run on windows? `defaultTargetPlatform == TargetPlatform.macOS` is false when I tested on my mac.

Reply:
> can this just be true? what will happen if someone tries to build devtools and run on windows?

If someone presses the import button in a platform that is not supported by file_selector then an [Unimplemented error][1] will be thrown.

> defaultTargetPlatform == TargetPlatform.macOS is false when I tested on my mac.
 
Oh then I should find another way to discover the platform the project is running on. 
 
[1]: https://github.com/flutter/plugins/blob/master/packages/file_selector/file_selector_platform_interface/lib/src/platform_interface/file_selector_interface.dart#L55

Reply:
Is there some reason you're not using `file_selector_windows`?

Reply:
> Is there some reason you're not using file_selector_windows?

That's a good point. Let me add it and remove the platform validation. @kenzieschmoll does that sounds good to you?

Reply:
making it work on all platforms sgtm 👍

Reply:
end users won't be hit by this anyway - they use DevTools on web. We mostly develop on Mac and Linux and none of us to my knowledge develop on windows.

Reply:
I just added windows and removed the platform validation. Thanks to both :)!